### PR TITLE
doc: nrf_security: Fix driver naming in configs

### PR DESCRIPTION
**Pull request by mswarowsky**
Fix some small issues in the naming of the drivers and a few other stuff

### CODE REVIEW

**Review by mia-ko — APPROVED** [review]

File: doc/nrf/libraries/nrf_security/doc/driver_config.rst [hunk 1]
```diff
@@ -28,9 +28,9 @@ To enable a PSA driver, set the configurations in the following table:
 
 If multiple drivers are enabled, the first ordered item in this table takes precedence for an enabled cryptographic feature, unless the driver does not enable or support it.
 
-The driver :ref:`nrf_security_drivers_cc3xx` allows enabling or disabling of specific PSA APIs (such as psa_cipher_encrypt, psa_sign_hash), but not individual algorithms.
+The :ref:`nrf_security_drivers_cc3xx` allows enabling or disabling of specific PSA APIs (such as psa_cipher_encrypt, psa_sign_hash), but not individual algorithms.
```
Comment (melwee01) [inline doc/nrf/libraries/nrf_security/doc/driver_config.rst:31]: The what? The article is now missing a noun.

Either it needs a noun or we take 'the' out altogether. Same applies to the next lines.

Comment (mia-ko) [reply doc/nrf/libraries/nrf_security/doc/driver_config.rst:31]: Mel, the link displays the section title where the noun 'driver' is shown. The article is with that in mind. Still want to remove the article?

Comment (mia-ko) [reply doc/nrf/libraries/nrf_security/doc/driver_config.rst:31]: image

Comment (mswarowsky) [reply doc/nrf/libraries/nrf_security/doc/driver_config.rst:31]: @mia-ko requested that, so maybe she can explain

**Review by endre-nordic — APPROVED** [review]

**Pull request merged**